assets/scss/base/layouts/_layouts.scss

```diff
@@ -96,3 +96,47 @@ body {
 .row {
   margin-bottom: 20px;
 }
+
+/*
+Half column container component
+
+Experimental: Half column component
+
+Markup:
+<div class="column-half-container">
+    <div class="grid-wrapper row-border">
+        <div class="grid grid-1-2">
+            <div>Some content</div>
+        </div>
+        <div class="grid grid-1-2">
+            <div>Some content</a>
+        </div>
+    </div>
+</div>
+
+.column-half-container - component containing two half column grids redefining parts of row-border and grid-wrapper elements
```

Comment:
no need for the modifier here as you haven't used the `{{modifier_class}}` variable.
The description here can be placed up above in the main description